packages/libs/wdk-client/src/Controllers/RecordController.tsx

```diff
@@ -111,7 +111,7 @@ class RecordController extends PageController<Props> {
           return (
             attributes?.includes(id) ||
             tables?.includes(id) ||
-            scopes.includes('record-internal')
+            scopes.includes('record-internal') // do we need to handle 'record-collapsed'?
```

Comment:
No, it isn't needed here. This code comes into play when attributes or tables are specified in the url, which is a way to have a record page with only those attributes and tables. Clearly this needs to be better documented.